addons/api/mirage/factories/managed-group.js

```diff
@@ -23,12 +24,18 @@ export default factory.extend({
 
   id: () => generateId('mg_'),
 
+  type: (i) => TYPES_AUTH_METHOD[i % TYPES_AUTH_METHOD.length],
+
   attributes() {
     switch (this.type) {
-      case 'oidc':
+      case TYPE_AUTH_METHOD_OIDC:
         return {
           filter: faker.random.words(),
         };
+      case TYPE_AUTH_METHOD_LDAP:
+        return {
+          group_names: [faker.internet.userName({ min: 1, max: 5 })],
```

Comment:
i think `faker.random.word` might work better here.  Also I don't think [userName](https://v7.fakerjs.dev/api/internet.html#username) has the min and max option

Reply:
@lisbet-alvarez good call, fixed